index.php

```diff
@@ -13,34 +13,17 @@
             <div class="navbar">
                 <a class="underline">Collection</a>
                 <a class="underline">Add Item</a>
-                <a class="underline">Edit Item</a>
+                <!-- <a class="underline">Edit Item</a> -->
             </div>
         </nav>
     <main>
         <?php
         require_once('getDbFunction.php');
         $db=getDb();
-        $query=$db->prepare("SELECT `brand`,`image`,`model`,`price`,`materials` FROM `bags`;");
-        $query->execute();
-        $result = $query->fetchAll();
-
-        echo '<div class="bags">';
-        foreach ($result as $bag) {
-            echo '<div class="item">';
-            echo '<h2>'.$bag['brand'].'</h2>';
-            echo '<br>';
-            echo '<div class="image">';
-            echo '<img src="'.$bag['image'].'">';
-            echo '</div>';
-            echo '<br>';
-            echo '<ul>';
-                echo '<li>'.$bag['model'].'</li><br>';
-                echo '<li>£'.$bag['price'].'</li><br>';
-                echo '<li>'.$bag['materials'].'</li>';
-            echo '</ul>';
-            echo '</div>';
-        };
-        echo '</div>';
+        require_once('getBagsFunction.php');
+        $bags=getBags($db);
+        require_once('listBagsFunction.php');
+        $collection=listBags($bags);
```

Comment:
This is great! My only comment is that it's considered best practice to put any requires or includes at the top of the file (above the HTML). It makes no real difference in terms of how the code works, but does mean that you can see any required files really quickly when you open the file.